content/en/tracing/profiler/getting_started.md

```diff
@@ -76,70 +76,103 @@ The Datadog Profiler requires [JDK Flight Recorder][1]. The Datadog Profiler lib
+```shell
+apk install gcc musl-dev linux-headers
+```
+
+**Usage**
+
+To automatically profile your code, set the `DD_PROFILING_ENABLED` environment
+variable to `true` when you use `ddtrace-run`:
 
-    ```
     DD_PROFILING_ENABLED=true ddtrace-run python app.py
-    ```
-    **Note:** `DD_PROFILING_ENABLED` is only supported in `dd-trace` version 0.40+. Use the alternate method if you are running an older version of `dd-trace`.
 
-    **Alternate method**
+It is strongly recommended that you add tags like `service` or `version`, as they
+provide the ability to slice and dice your profiles across these dimensions,
+enhancing your overall product experience. See [Configuration][#configuration] below.
 
-    If you prefer to instrument the profiler through code, import `ddtrace.profile.auto`. After import, the Profiler starts:
+After a couple of minutes, visualize your profiles on the [Datadog APM > Profiler page][2].
 
-    ```python
-    import ddtrace.profiling.auto
-    ```
+If you want to manually control the lifecycle of the profiler, use the
+`ddtrace.profiling.profiler.Profiler` object:
 
-4. After a minute or two, visualize your profiles on the [Datadog APM > Profiler page][2].
+```python
+from ddtrace.profiling import Profiler
 
-- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use environment variables to set the parameters:
+prof = Profiler()
+prof.start()
 
-| Environment variable                             | Type          | Description                                                                                      |
-| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
-| `DD_PROFILING_ENABLED`                           | Boolean       | Set to `true` to enable profiler. Supported from tracer version 0.40+.              |
-| `DD_SERVICE`                                     | String        | The Datadog [service][3] name.     |
-| `DD_ENV`                                         | String        | The Datadog [environment][4] name, for example, `production`. |
-| `DD_VERSION`                                     | String        | The version of your application.                             |
-| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api,team:intake`.   |
+# At shutdown
+prof.stop()
+```
 
-<div class="alert alert-info">
-Recommended for advanced usage only.
+<div class="alert alert-warning">
+The profiler has been designed to be turned on continuously. Only use the
+`start` and `stop` methods to start and stop the profiler at the beginning and
+end of your program, not to selectively profile parts of your program.
```

Comment:
If we do the above, we can probably ditch this. The less stuff people have to think about when onboarding, the better!